authors/apps/articles/tests/test_get_profiles.py

```diff
@@ -0,0 +1,75 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from authors.apps.articles.models import Articles
+from authors.apps.authentication.models import User
+
+PROFILES_URL = reverse('articles:author-profile-list')
+
+class ArticleTest(TestCase):
```

Comment:
Change this class name to something related to profiles like --> ProfileTest

Reply:
done and done